archunit/src/test/java/com/tngtech/archunit/lang/ArchRuleTest.java

```diff
@@ -169,32 +168,43 @@ public void finish(ConditionEvents events) {
     }
 
     @Test
-    public void evaluation_fails_because_of_empty_set_of_classes_with_default_fail_on_empty_should() {
-        archConfigurationRule.removeProperty(FAIL_ON_EMPTY_SHOULD_PROPERTY_NAME);
-
+    public void evaluation_fails_because_of_empty_set_of_elements_with_default_fail_on_empty_should() {
         thrown.expect(AssertionError.class);
         thrown.expectMessage("Rule failed to check any classes");
         thrown.expectMessage(FAIL_ON_EMPTY_SHOULD_PROPERTY_NAME);
 
-        classes().should(ALWAYS_BE_VALID).evaluate(importClasses());
+        createPassingArchRule().evaluate(importClasses());
```

Comment:
_Unrelated to the changes in this PR:_ I was confused why this fails, until I realized that `importClasses()` actually imports no classes. 🙈 
We could use `importClasses(/* no classes */)` or even have a semantic method / variable.

Reply:
Changed to separate method :wink: